### PR TITLE
polish weights

### PR DESCRIPTION
**Pull request by heavengate**
**polish weights**
- convert remain static weights to dynamic after https://github.com/PaddlePaddle/PaddleDetection/pull/2366
- add `backbone.` prefix in backbone pretrain weights and remove `--weight-type` in `tools/train.py`
- add `--resume` for weight resuming in `tools/train.py`
- add `ppdet://` as alias for `https://paddledet.bj.bcebos.com/` to simplify URLs

### CODE REVIEW

File: dygraph/configs/faster_rcnn/_base_/faster_rcnn_r50.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 architecture: FasterRCNN
-pretrain_weights: https://paddledet.bj.bcebos.com/models/pretrained/ResNet50_cos_pretrained.pdparams
+pretrain_weights: ppdet://models/pretrained/ResNet50_cos_pretrained.pdparams
```
Comment (qingqing01) [inline dygraph/configs/faster_rcnn/_base_/faster_rcnn_r50.yml]: 原先的完整链接不好吗？

Comment (heavengate) [reply dygraph/configs/faster_rcnn/_base_/faster_rcnn_r50.yml]: 使用简化链接用户想通过`-o weights=`或者`wget`获取预训练模型或者model zoo模型比较方便，配置文件也简洁一些，竞品也普遍采用简化链接的方式

**Review by jerrywgz — COMMENTED** [review]
eval.py,infer.py, export_model.py等入口的load_weights也需要适配

**Comment by heavengate** [comment]
> eval.py,infer.py, export_model.py等入口的load_weights也需要适配

Done, thanks!

File: dygraph/configs/faster_rcnn/_base_/faster_rcnn_r50.yml (hunk 1 above)
Comment (heavengate) [reply dygraph/configs/faster_rcnn/_base_/faster_rcnn_r50.yml]: Done, thanks!

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**